### ISSUE TITLE
If aux_date is unspecified, actual date should be used (BZ#1157)

### ISSUE THREAD

**Issue by tbm** [ledger/ledger#1157]
Note: the issue was created automatically with bugzilla2github

Bugzilla bug ID: BZ#1157
From: Tobias Pfeiffer (@tgpfeiffer)

**Comment by Fuco1** [comment]
I think maybe explicitly writing the formula would be better

```
= expr account =~ /Liabilities/ and (aux_date ? aux_date : date) < [2014/11]
```

Theoretically for someone it might be an error to not have aux_date specified and I *feel* like we shouldn't magically fill it with `date`.  The absence of a value sometimes is also valuable information.

WDYT?

**Comment by jwiegley** [comment]
I agree that the more complex expression is the right answer here, rather than making `aux_date` magical in this way.
